jdaviz/configs/cubeviz/helper.py

```diff
@@ -121,6 +121,31 @@ def specviz(self):
             self._specviz = Specviz(app=self.app)
         return self._specviz
 
+    def get_data(self, data_label=None, cls=None, subset_to_apply=None, function=None):
+        """
+        Returns data with name equal to data_label of type cls with subsets applied from
+        subset_to_apply.
+
+        Parameters
+        ----------
+        data_label : str, optional
+            Provide a label to retrieve a specific data set from data_collection.
+        cls : `~specutils.Spectrum1D`, `~astropy.nddata.CCDData`, optional
+            The type that data will be returned as.
+        subset_to_apply : str, optional
+            Subset that is to be applied to data before it is returned.
+        function : {'minimum', 'maximum', 'mean', 'median', 'sum'}, optional
```

Comment:
If there is a default that grabs from Plot Options (I am guessing that is `None`?), should be documented here.